src/main/java/staffconnect/logic/commands/AddMeetingCommand.java

```diff
@@ -28,6 +28,7 @@ public class AddMeetingCommand extends Command {
             + "by the index number used in the displayed person list. \n"
             + "Parameters: INDEX (must be a positive integer) " + PREFIX_MEETING_DESCRIPTION + "DESCRIPTION "
             + PREFIX_MEETING_STARTDATE + "DATETIME[dd/MM/yyyy HH:mm] \n"
+            + "There are variant ways to type in a start date. The above one is recommended.\n"
```

Comment:
Typo and sentence structure issue. 
Suggested modification:
`There are various ways to type the start date. The one above is recommended.\n`